src/Components/UserProfileCard.js

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
-import { Button, Card, Icon, Image } from 'semantic-ui-react';
+import { Grid, Button, Card, Icon, Image } from 'semantic-ui-react';
 
 const UserProfileCard = () => {
   return (
     <Card centered>
```

Comment:
The card here needs to have the fluid prop so it takes the full space of the container.